tensorbay/cli/cli.py

```diff
@@ -92,21 +92,23 @@ def cli(ctx: click.Context, access_key: str, url: str, profile_name: str, debug:
 @click.option(
     "-a", "--all", "list_all_files", is_flag=True, help="List all files under the segment."
 )
+@click.option("-l", "--list", "show_total_num", is_flag=True, help="Show the total number of files")
```

Comment:
is `--list` necessary, can we only support `-l` because linux `ls` only supports `-l` and not supports `--list`